x-pack/plugins/index_management/public/application/services/breadcrumbs.ts

```diff
@@ -70,11 +78,21 @@ class BreadcrumbService {
       },
     ];
 
-    this.breadcrumbs.enrichPolicyCreate = [
+    this.breadcrumbs.enrichPolicies = [
       ...this.breadcrumbs.home,
       {
-        text: i18n.translate('xpack.idxMgmt.breadcrumb.createEnrichPolicyLabel', {
-          defaultMessage: 'Create enrich policy',
+        text: i18n.translate('xpack.idxMgmt.breadcrumb.enrichPolicyLabel', {
+          defaultMessage: 'Enrich policies',
+        }),
+        href: `/enrich_policies`,
+      },
+    ];
+
+    this.breadcrumbs.enrichPoliciesCreate = [
+      ...this.breadcrumbs.enrichPolicies,
+      {
+        text: i18n.translate('xpack.idxMgmt.breadcrumb.enrichPolicyCreateLabel', {
+          defaultMessage: 'Create enrich policies',
```

Comment:
[nit] Should the breadcrumb copy align with the page title?

```suggestion
          defaultMessage: 'Create enrich policy',
```

<img width="651" alt="Screenshot 2023-09-18 at 12 51 38 PM" src="https://github.com/elastic/kibana/assets/5226211/9f48a27c-5324-4943-bb93-691981f7311c">